pkg/ui/page/projects.go

```diff
@@ -127,11 +169,16 @@ func (p *ProjectList) fillProjectList(ctx *controller.ProjectListPageContext) {
 
 	// Set body.
 	rowPosition := 1
+	filter := p.getFilter()
 	for _, project := range ctx.Projects {
 		if project == nil {
 			continue
 		}
 
+		if filter != "" && strings.Contains(strings.ToLower(project.Name), filter) == false {
```

Comment:
What about filtering by a  regex? thoughts?

Reply:
Will try to implement this with regular expressions! :+1: